inc/t_cose/t_cose_common.h

```diff
@@ -559,7 +592,12 @@ enum t_cose_err_t {
     /* A newer version of QCBOR is needed to processes multiple
      * COSE_Signature or COSE_Recipients.  (As of Jan 2023, this
      * QCBOR is not released) */
-    T_COSE_ERR_CANT_PROCESS_MULTIPLE = 71
+    T_COSE_ERR_CANT_PROCESS_MULTIPLE = 71,
+
+    /* A signing operation is in progress. The function returning this value
+     * can be called again until it returns \ref T_COSE_SUCCESS or error.
+     */
+    T_COSE_ERR_SIG_IN_PROGRESS = 72,
```

Comment:
Not sure why there's no merge conflict here yet. There will be because of other errors defined. But a simple merge.